CHANGELOG.md

```diff
@@ -24,6 +24,28 @@ CHANGELOG
 Swift 5.0
 ---------
 
+* [SE-0235][]:
+
+  The standard library now contains a `Result<Success, Failure: Error>` type to 
+  manually model potentially error-producing code.
+  
+  ```swift
+  enum Result<Success, Failure: Error> {
+      case success(Success)
+      case failure(Failure)
+  }
+  ```
+  
+  This type serves a separate but complementary role to Swift's existing and 
```

Comment:
```suggestion
  This type serves a complementary role to that of `throws`.
```

"Complementary" implies "separate"; and you do not want to echo the phrase "separate but equal."  
"Swift" is unnecessary because it is implied.  
"Existing" is unnecessary because one would hardly bring up non-existent features for comparison.